components/oneapi-validation/src/main/java/com/wso2telco/dep/oneapivalidation/service/impl/ussd/ValidateUssdSubscription.java

```diff
@@ -63,14 +65,25 @@ public void validate(String json) throws CustomException {
                 if (!callbackReference.isNull("notifyURL")) {
                     notifyUrl = nullOrTrimmed(callbackReference.getString("notifyURL"));
                 }
+            } else {
+                throw new CustomException("SVC0002", "Invalid input value for message part %1",
+                        new String[]{"Missing mandatory parameter: callbackReference"});
             }
 
-            ValidationRule[] rules = null;
+            if (requestData.has(USSDKeyConstants.SHORT_CODE) && !requestData.isNull(USSDKeyConstants.SHORT_CODE)) {
```

Comment:
No need of has() check
Add HUB validation rules for shortCode and keyword

Reply:
done